py/picca/delta_extraction/expected_fluxes/dr16_expected_flux.py

```diff
@@ -760,8 +761,8 @@ def chi2(eta, var_lss, fudge):
                     variance)
                 weights = var2_delta[index * num_var_bins:(index + 1) *
                                      num_var_bins]
-                w = num_qso[index * num_var_bins:(index + 1) *
-                            num_var_bins] > 100
+                w = count_qso[index * num_var_bins:(index + 1) *
+                            num_var_bins] > 0
```

Comment:
can we make the min_num_qso a variable of the cont fitter? So that previous results are still reproducable if the variable is set to 100. And tests will still give the same result as before if 100 is used.

Can we then have the actual change to 0 in a seperate PR that does only this and a change of tests? I think that would make clearer which exact change produced changes in test results.

Reply:
sure, I'll do another PR with changing it to a variable and then will update this one with the change to 0

Reply:
I merged that PR to master and here, what is missing is only the new default value or value in the test config...